include/picongpu/fields/currentDeposition/EmZ/DepositCurrent.hpp

```diff
@@ -69,10 +70,10 @@ namespace picongpu
                  * @param line trajectory of the virtual particle
                  * @param currentSurfaceDensity surface density
                  */
-                template<typename CursorJ, typename T_Line, typename T_Worker>
+                template<typename T_DataBox, typename T_Permutation, typename T_Line, typename T_Worker>
```

Comment:
The added `T_Permutation` is missing in above doxygen description

Reply:
@psychocoderHPC ping

Reply:
added some doc strings but skipped `T_Wroker` because this is a general PMacc concept